notebooks/book2/23/flow_2d_mlp.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        ax.scatter(x[:, 0], x[:, 1], s=10, cmap=cm, c=color)
</pre><p><code>s = 10</code> is too much for latexified figure, can you decrease it so we can see proper scatters in latexified figure. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
After keeping s=1 I obtained this plot: 
<img width="714" alt="image" src="https://user-images.githubusercontent.com/76394914/177028192-c00407f5-1756-419b-b94e-1fc364753f50.png">
Should I go ahead with this?

Reply:
scatter size looks good but in the first sub-plot `xtick_labels` are too closer. You can fix is setting only [-2.5, 2.5] or [-2, 0, 2] for first subplot only.

Reply:
After resolving this, you can directly create a PR in main repo.